src-tool/Pact/Analyze/Check.hs

```diff
@@ -254,11 +255,13 @@ describeSmtFailure = \case
   UnexpectedFailure smtE -> T.pack $ show smtE
 
 describeUnknownFailure :: SBV.SMTReasonUnknown -> Text
-describeUnknownFailure reason = "You've written a hell of a property here. Usually properties are simple things, like \"is positive\" or \"conserves mass\". But not this bad boy. This here property broke the SMT solver. Wish we could help but you're on your own with this one (actually, please report this as an issue: https://github.com/kadena-io/pact/issues).\n\nGood luck...\n" <> tShow reason
+describeUnknownFailure r = case show r of
+  "timeout" -> "SMT solver timeout"
+  _ ->  "Unexpected failure from SMT solver: " <> tShow r
```

Comment:
`sbv` has an `SMTReasonUnknown` case for this called `UnknownTimeOut`: https://hackage.haskell.org/package/sbv-8.15/docs/Data-SBV-Trans.html#t:SMTReasonUnknown.